### ISSUE TITLE
Check use of the `#{pattern}`s in the locales

### ISSUE THREAD

**Issue by ST-DDT** [faker-js/faker#725]
### Describe the bug

A lot of locale files such as https://github.com/faker-js/faker/blob/main/src/locales/en/address/city.ts contain a `#{pattern}`

These files aren't used anywhere and faker does not have a method to use/resolve them with.

Did `faker.fake()` support this syntax at some point or have they been forgotten during a refactor related to `faker.fake()`?

I'm not sure what to do with them.

### Reproduction

https://github.com/faker-js/faker/blob/2532eb9f5a6f73bbc1786ec91952f838d98fd72a/src/locales/en/address/city.ts#L1-L6

### Additional Info

There are almost 200 locale files that contain these.

**Comment by Shinigami92** [comment]
Never recognized this `#{` before :thinking:

**Comment by ST-DDT** [comment]
I traced it back to this commit:
https://github.com/faker-js/faker/blob/0048b9f6cfd69378ca3ba2f588444e7bb8095d23/lib/locales/en.js#L1027-L1032
> [api] First pass at adding localization data definitions. #116 (14 Sep 2014)
which was included in v2.0.0

I searched a few files and commits, but didn't find any method ever dealing with this syntax.

**Comment by cxcorp** [comment]
I'm not sure if this is related, but the `en_GH` locale seems to produce a lot of strings with those in them:

```js
faker.locale = 'en_GH';
faker.internet.exampleEmail();
// 'male_first_name22@example.com'
faker.internet.userName()
// '#{female_first_name}.Akowua'
```

**Comment by ST-DDT** [comment]
https://github.com/faker-js/faker/blob/main/src/locales/en_GH/name/first_name.ts
https://github.com/faker-js/faker/blob/main/src/locales/en/name/first_name.ts

In this case its more like a bug. The file contains bad data.
You could create a PR that imports both male and female, sorts and exports them instead

**Comment by ST-DDT** [comment]
Will hopefully be fixed by #927
